CircularSaw.java

```diff
@@ -0,0 +1,34 @@
+package ua.lviv.iot.algo.part1.lab1;
+import lombok.*;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Setter
+@Getter
+@ToString
+
+public class CircularSaw extends Saw{
+    private double radius;
+    private double width;
+    public boolean start(){
+        setWorking(true);
+        return getWorking();
+    }
+    public boolean stop(){
+        setWorking(false);
+        return getWorking();
+    }
+    public String convertingToString(){
```

Comment:
name of the method should start with infinitive verb, better change it to "convert" without ing